lib/asset-s3-stat.js

```diff
@@ -307,7 +333,7 @@ s3stat.Task.prototype._parseUploadFiles = function() {
 
     sourceFile = grunt.file.expand(assetObj.absPath)[0];
 
-    if (!sourceFile) {
+    if (!sourceFile || !assetObj.absPath.match(/(app\.min|ie\.min)/)) {
```

Comment:
Oh crap, that was not supposed to be here. That was just a temporary workaround for our project